www/src/pages/en/faq.mdx

```diff
@@ -21,6 +21,7 @@ If you are not familiar with the different technologies used in this project, pl
 - [Prisma](https://prisma.io)
 - [Tailwind CSS](https://tailwindcss.com)
 - [tRPC](https://trpc.io)
+- [Drizzle](https://drizzle.team/)
```

Comment:
If create-t3-app is using drizzle-orm itself, I think it would be better to have a link to drizzle-orm docs rather than the drizzle team. I guess this one should be much better: https://orm.drizzle.team/docs/overview. We have also updated an overview page and the documentation itself.